### ISSUE TITLE
Add linter and formatter to the root folder package.json

### ISSUE THREAD

**Issue by Mikearaya** [unchainedshop/create#85]
Currently, there are only helper scripts are added in the root folder package.json when a user creates a full-stack template.
there should be eslint/prettier linter/formater packages included by default so that the environment is set and ready to do actual development out of the box.
it would also be great if, after injecting these packages the script installs these devDependencies   before completing the template generation.

 so basically:
[ ] Add linter and formaters to package.json
[ ] install this dependency
[ ] Add a script to use the linter and formater

**Comment by pozylon** [comment]
But the subdirectories have appropriate linter and formatter configs already? @Mikearaya
